pkg/crc/machine/machine.go

```diff
@@ -195,11 +196,32 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		logging.InfoF("Starting OpenShift cluster ... [waiting 3m]")
 	}
 	result.KubeletStarted = kubeletStarted
-	//
 
 	// If no error, return usage message
 	if result.Error == "" {
 		time.Sleep(time.Minute * 3)
+		// Checks after 3 mins wait all the operators are in 'Available' state.
+		// If any of the operator still not available then wait for 2 more mins and tell users
+		// about which operator is still not available and need time.
+		var status bool
+		for i := 0; i < 2; i++ {
+			status, err = oc.GetClusterOperatorStatus(filepath.Join(constants.MachineCacheDir, "oc", "oc"), filepath.Join(extractedPath, crcBundleMetadata.ClusterInfo.KubeConfig))
```

Comment:
can we create the command as a separate variable?
and same for the kubeconfig path?

These can be reused, and as varibale it makes the code more readable.